lib/reporting/reports/orders_and_fulfillment/order_cycle_customer_totals.rb

```diff
@@ -163,6 +167,23 @@ def distributor_customer(line_items)
           user = line_items.first.order.user
           user&.customer_of(distributor)
         end
+
+        def voucher_label(order)
+          return '' unless voucher_applicable?(order)
+
+          voucher = order.voucher_adjustments.take.originator
+          voucher&.code.to_s # in case if we don't get the voucher, return ""
```

Comment:
Given that we know there has been a voucher applied to the order, we shouldn't need the `&` here.

```suggestion
          voucher.code.to_s
```